ui/app/src/views/PoolPage.vue

```diff
@@ -2,34 +2,68 @@
 import { defineComponent } from "vue";
 
 import Layout from "@/components/layout/Layout.vue";
+import PoolList from "@/components/poolList/PoolList.vue";
+import PoolListItem from "@/components/poolList/PoolListItem.vue";
+import SifButton from "@/components/shared/SifButton.vue";
+import PriceCalculation from "@/components/shared/PriceCalculation.vue";
+
 export default defineComponent({
-  components: { Layout },
+  components: { Layout, SifButton, PriceCalculation, PoolList, PoolListItem },
 });
 </script>
 
 <template>
-  <Layout class="pool">
-    <router-link class="button" to="/pool/create-pool">Create Pool</router-link>
-    <router-link class="button" to="/pool/add-liquidity"
-      >Add Liquidity</router-link
-    >
-    <router-link class="button" to="/pool/remove-liquidity"
-      >Remove Liquidity</router-link
-    >
+  <Layout>
+    <div>
+      <div class="heading mb-8">
+        <h3>Your Liquidity</h3>
+        <router-link to="/pool/create-pool" class="pr-4"
+          ><SifButton primaryOutline nocase>Create Pair</SifButton></router-link
+        >&nbsp;
+        <router-link to="/pool/add-liquidity"
+          ><SifButton primary nocase>Add Liquidity</SifButton></router-link
+        >
+      </div>
+      <div class="mb-8">
+        <SifButton primaryOutline nocase block>Account analytics and accrued fees</SifButton>
+      </div>
+      <PriceCalculation class="mb-8">
+        <div class="info">
+          <h3 class="mb-2">Liquidity provider rewards</h3>
+          <p class="text--small mb-2">Liquidity providers earn a 0.3% fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.</p>
+          <p class="text--small mb-2"><a href="#">Read more about providing liquidity</a></p>
+        </div>
+      </PriceCalculation>
+      <PoolList class="mb-2">
+        <PoolListItem />
+        <PoolListItem />
+        <PoolListItem />
+        <PoolListItem />
+      </PoolList>
+      <div class="footer">
+        Don’t see a pool you joined? <a href="#">Import it</a>
+      </div>
+    </div>
   </Layout>
 </template>
 
-<style scoped>
-.button {
-  display: block;
-  padding: 1rem;
-  text-decoration: none;
-  color: #333;
-  background: #aaa;
-  margin-bottom: 1rem;
+<style scoped lang="scss">
+.heading {
+  display: flex;
+  align-items: center;
+
+  h3 {
+    @include title16;
```

Comment:
Have we got an explanation of these different titles etc. anywhere? Would love to see a styleguide page or our components displayed in storybook.

Reply:
Not yet. There are a lot of different copy/type styles and with the new designs, new styles are emerging. I'll look into the best solution to organize these.